design/20190708.certificate-request-crd.md

```diff
@@ -427,50 +433,21 @@ this resource.
 
 #### Issuing Controller
 
-Since external issuers have been built before the addition of Approved and
-Denied conditions, the issuing controller needs to be permissive. An external
-issuer may not honour an Approved condition and will sign and set a
-CertificateRequest as being Ready, before the request has been approved. The
-issuing controller must mark issuance as being successful in this case. In
-practice, this means that the issuing controller is never concerned with
-Approved conditions.
-
-External issuers that do not honour Denied conditions will sign
```

Comment:
The behaviour most likely wouldn't have been like this actually due to the race condition and since we've had it like so for almost two years I assume it is fine to consistently not do this